lib/workload/stateless/icav2_copy_batch_utility/step_functions_templates/copy_batch_state_machine.json

```diff
@@ -0,0 +1,188 @@
+{
+  "Comment": "Deploy a set of ICAv2 copy batch jobs and wait until theyre finished",
```

Comment:
Btw Alexis @alexiswl, I'd recommend to name these ASL into the naming convention `copy_batch_state_machine.asl.json`

There is benefit in IDE that support ASL syntax and, able to render step diagram. 

cc: @reisingerf

Reply:
Oh no way! I was wondering if there was a JSON schema for this, thanks @victorskl, I'll do that now

Reply:
Bad news is - only avail in VSCode and draw.io at the mo. 😢 

Watch this space
https://github.com/aws/aws-toolkit-jetbrains/issues/584